src/test/mocks/fetch/index.ts

```diff
@@ -36,6 +36,7 @@ const mockFetch = jest.fn((input, _init?) => {
   });
 });
 
+// TODO: need to mock with `msw`
 global.fetch = mockFetch as any;
```

Comment:
Actually, `fetch` is mocked as a regular function but it should be properly mocked as an HTTP handler using `msw`.
I'm going to handle it as well when mocking the squid endpoint.